frame/staking/src/pallet/impls.rs

```diff
@@ -719,6 +722,7 @@ impl<T: Config> Pallet<T> {
 		let mut voters_seen = 0u32;
 		let mut validators_taken = 0u32;
 		let mut nominators_taken = 0u32;
+		let mut min_active_stake = u64::MAX;
```

Comment:
I would have preferred to give it a type Option<u64> and initialize it to `None`.

Reply:
Never mind, saw Kian's comment :joy: